src/commands/deploy.ts

```diff
@@ -35,7 +35,7 @@ export const deployCommand = new Command()
             console.log(`Plugin ${id} updated successfully.`);
         } catch (error) {
             console.log('Plugin not found. Attempting to register...');
-            const result = await registerPlugin(id, accountId);
+            const result = await registerPlugin({ pluginId: id, accountId });
```

Comment:
I thought [this](https://github.com/BitteProtocol/make-agent/pull/17/files#diff-5ebd967f545152006c709d66e65827c90a402b4d72228a3e2c7abe2c11abe667R96) was the fix since it was incorrectly sending the bitte key object instead of the account id.

Reply:
Yes, great catch.  Commented on the wrong file entirely.  Also [this was changed](https://github.com/BitteProtocol/make-agent/pull/17/files#diff-5ebd967f545152006c709d66e65827c90a402b4d72228a3e2c7abe2c11abe667L50-L51:~:text=%3A%20await%20registerPlugin(%7B%20pluginId%2C%20accountId%20%7D)%3B) to include accountId

Reply:
Aight, I'm sending it 🚀